query-service-impl/src/main/java/org/hypertrace/core/query/service/ExecutionContext.java

```diff
@@ -275,6 +270,6 @@ public ResultSetMetadata getResultSetMetadata() {
   }
 
   public Duration getTimeRangeDuration() {
-    return this.timeRangeDuration;
+    return this.timeRangeDuration.get();
```

Comment:
Return the optional. Optionals are all about the return - we gain nothing by storing it as an optional then .get()'ing it - it's not optional, since this would throw an exception if it was missing.